extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.js

```diff
@@ -239,29 +242,69 @@ export class AmpLightboxViewer extends AMP.BaseElement {
       this.descriptionBox_.classList.add('overflow');
       this.topBar_.classList.add('overflow');
       this.vsync_.run({
-        measure: state => {
-          state.descTextAreaHeight =
+        measure: state1 => {
+          state1.prevDescTextAreaHeight =
               this.descriptionTextArea_./*OK*/scrollHeight;
-          state.descBoxHeight = this.descriptionBox_./*OK*/clientHeight;
+          state1.descBoxHeight = this.descriptionBox_./*OK*/clientHeight;
         },
-        mutate: state => {
-          if (state.descTextAreaHeight < state.descBoxHeight) {
-            const diffHeight = state.descBoxHeight - state.descTextAreaHeight;
-            setStyle(this.descriptionTextArea_, 'top', `${diffHeight}px`);
-          }
+        mutate: state1 => {
+          this.descriptionTextArea_.classList.remove('non-expanded');
+          const tempDiffHeight =
+              state1.descBoxHeight - state1.prevDescTextAreaHeight;
+          setStyle(this.descriptionTextArea_, 'top', `${tempDiffHeight}px`);
+          this.vsync_.run({
```

Comment:
let's move the code inside this `run()` to a constant outside for better readability i.e. something like `this.vsync_.run(animate)`